### ISSUE TITLE
Feature Request: Create ConfigSlider component in Ant Design

### ISSUE THREAD

**Issue by SanjaySinghRajpoot** [moja-global/FLINT-UI#176]
### Is your feature request related to a problem? Please describe.

Migrating old tailwind components to Ant Design. 

### Describe the solution you'd like.

This component will render the above Slider component with labels (Pool 1, Pool 2, etc.) and their current value (and max value, which is added in the new design)
There is already a component made for this but it uses the Slider from vueform. So this component can be replaced with a new one which uses the above Slider component.
(The old ConfigSlider.vue component is not used anywhere, so it is safe to be removed).

Props:
min - min value of the slider (default 0)
max - max value of the slider (default 400)
step - step value of the slider (default 5)
v-model from the parent will be given to this component since the state needs to be in the parent because it is used in the Run method.

### Additional context.

![Screenshot from 2022-03-11 11-26-11](https://user-images.githubusercontent.com/67458417/157810791-c9b655a7-809d-4221-af2a-21ce527e3934.png)

**Comment by SanjaySinghRajpoot** [comment]
Update I have to keep this issue on hold. Once the Slider #175 issue is fixed. Then we can start working on this issue. Till then please work on some other issue. 

Thank You.

**Comment by Palaksharma23** [comment]
@chicken-biryani maam I would like to work on this issue :)

**Comment by chicken-biryani** [comment]
Yes, @Palaksharma23 now that the prev issue has been worked upon you can work on this.

Do keep us updated with your progress. :)

**Comment by Palaksharma23** [comment]
sure mam :)

**Comment by GulabSinghSikarwar** [comment]
@SanjaySinghRajpoot , Sir, I  want to start my Open Source Journey, By Contributing to This issue

**Comment by chicken-biryani** [comment]
Hi @Palaksharma23 any updates?

@GulabSinghSikarwar I believe Palak is already working on it.

**Comment by Palaksharma23** [comment]
maam let @GulabSinghSikarwar give it a try :)

**Comment by chicken-biryani** [comment]
@Palaksharma23 okay.

@GulabSinghSikarwar please take over. And keep us updated.

**Comment by GulabSinghSikarwar** [comment]
Ok Mam

**Comment by sohamsshah** [comment]
Hi @GulabSinghSikarwar , any updates on the issue? Please let me know if you need any help or doubts around it's implementation

**Comment by chicken-biryani** [comment]
@YashKandalkar @Palaksharma23 is this issue to be worked upon or can we close it?

**Comment by Palaksharma23** [comment]
I assume the slider we currently have is fine. After some months, we will be able to directly import it from our UI library. We can close it for now @chicken-biryani :).
